src/pages/320x240x16/voiceconfig_page.c

```diff
@@ -32,12 +32,10 @@ enum {
 static int row_cb(int absrow, int relrow, int y, void *data)
 {
     (void)data;
-        int row = y;
-        int voice_num = absrow;
         //Row 1
-        GUI_CreateLabelBox(&gui->name[relrow], COL1, row, COL2-COL1, 18, &LABEL_FONT,voiceconfig_str_cb, NULL, (void *)(long)voice_num);
-        GUI_CreateTextSelect(&gui->voiceidx[relrow],  COL2, row, TEXTSELECT_64, voice_test_cb, voiceid_cb, (void *)(long)voice_num);
-        GUI_CreateLabelBox(&gui->voicelbl[relrow], COL3, row, LCD_WIDTH - COL3 - ARROW_WIDTH, 18, &TINY_FONT, voicelbl_cb, NULL, (void *)(long)voice_num);
+        GUI_CreateLabelBox(&gui->name[relrow], COL1, y, COL2-COL1, 18, &LABEL_FONT, voiceconfig_str_cb, NULL, (void *)(long)absrow);
+        GUI_CreateTextSelect(&gui->voiceidx[relrow],  COL2, y, TEXTSELECT_64, voice_test_cb, voiceid_cb, (void *)(long)absrow);
+        GUI_CreateLabelBox(&gui->voicelbl[relrow], COL3, y, LCD_WIDTH - COL3 - ARROW_WIDTH, 18, &TINY_FONT, voicelbl_cb, NULL, (void *)(long)absrow);
```

Comment:
Lint Error:
```
36:  Use int16/int64/etc, rather than the C type long  
37:  Use int16/int64/etc, rather than the C type long  
38:  Use int16/int64/etc, rather than the C type long  
```